docs/apache-airflow/howto/operator/python.rst

```diff
@@ -239,3 +239,11 @@ should return True when it succeeds, False otherwise. The other uses the Taskflo
     :dedent: 4
     :start-after: [START wait_function]
     :end-before: [END wait_function]
+
+You can also use the @task.sensor decorator from TaskFlow to define your sensor as a TaskFlow sensor.
+
+.. exampleinclude:: /../../airflow/example_dags/example_taskflow_sensor.py
```

Comment:
I think this example does not exist.

Reply:
I have made that file in other PR....look in pull request

Reply:
Can you just combine this change in that other PR then? Thanks.

Reply:
sure

Reply:
@jedcunningham @pankajastro check this PR where I merged this into other one -> #30749

Reply:
then please close this PR in favour of https://github.com/apache/airflow/pull/30749